sysprep/sysprep.ps1

```diff
@@ -165,6 +172,10 @@ try {
     }
   }
 
+  if ($no_shutdown) {
```

Comment:
Let's add a comment so it's clear in the logs that we did not shut down due to user specified configuration.

Reply:
Done